README.md

```diff
@@ -37,6 +37,7 @@ The educandu framework
  | importSources | The system from which data can be imported | `[{ name, hostName, allowUnsecure, apiKey }]` | no, if not specified then `allowUnsecure` defaults to `false` |
  | disabledFeatures | A list of names of disabled features | `string[]` | no |
  | taskProcessing | Task processing setup | `{ isEnabled, idlePollIntervalInMs, maxAttempts }` | no, defaults to `{ isEnabled: false, idlePollIntervalInMs: 5000, maxAttempts: 3 }` |
+ | additionalControllers | Any custome controllers you need | arrayOfControllers: [] | no, defaults to [] |
```

Comment:
Small thing: I'd suggest to not introduce personal addressing phrasing and keep it impersonally phrased.
Maybe more like " | additionalController | Custom controllers | array of controller classes | no, defaults to `[]` "

Reply:
Okay, good point! Will fix

Reply:
Fixed